src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write your code here
+document.addEventListener('DOMContentLoaded', () => {
+  const populationElements = document.querySelectorAll('.population');
+  const populations = Array.from(populationElements)
+    .map((el) => parseInt(el.textContent.replace(/,/g, ''), 10))
+    .filter((num) => !isNaN(num));
+
+  const totalPopulation = populations.reduce((acc, num) => acc + num, 0);
+  const averagePopulation = Math.round(totalPopulation / populations.length);
```

Comment:
There is a potential issue here if `populations.length` is zero, which would cause a division by zero error. Consider adding a check to ensure `populations.length` is not zero before performing the division.